h/static/scripts/controllers/search-bar-controller.js

```diff
@@ -203,13 +241,15 @@ class SearchBarController extends Controller {
         this.setState({open: false});
       }
       closeDropdown();
+      this._input.removeEventListener('keydown', setupListenerKeys,
```

Comment:
I am not sure what you are expecting this to do. It's not actually going to remove the listener nor invoke setupListenerKeys.

See: https://jsfiddle.net/sean_roberts/swmhkjtm/